cpp/src/arrow/engine/substrait/serde.h

```diff
@@ -202,6 +207,18 @@ Result<std::shared_ptr<Buffer>> SerializeExpression(
     const compute::Expression& expr, ExtensionSet* ext_set,
     const ConversionOptions& conversion_options = {});
 
+/// \brief Serializes an Arrow compute Declaration to a Substrait Relation message
+///
+/// \param[in] declaration the Arrow compute declaration to serialize
+/// \param[in,out] ext_set the extension mapping to use; may be updated to add
```

Comment:
I'm not sure what `; may be updated to add` means.

Reply:
the latter part was not added. I added the full description. Thanks for noting this typo.